whois-db/src/main/java/net/ripe/db/whois/db/WhoisServer.java

```diff
@@ -102,4 +104,18 @@ private void stopService(final ApplicationService applicationService, boolean fo
             LOGGER.error("Stopping: {}", applicationService, e);
         }
     }
+
+    private static void printJvmSecurityProperties() {
+        if(!Boolean.valueOf(Security.getProperty("security.overridePropertiesFile"))) {
+            LOGGER.warn("security.overridePropertiesFile is false, cannot override security values");
+        }
+
+        final String networdkAddrCacheTtl = Security.getProperty("networkaddress.cache.ttl");
+        LOGGER.info("networkaddress.cache.ttl: {}", networdkAddrCacheTtl);
+        LOGGER.info("networkaddress.cache.negative.ttl: {}", Security.getProperty("networkaddress.cache.negative.ttl"));
```

Comment:
Also create a variable for the negative TTL value and log separately

Reply:
I am logging the value for negative ttl, there is no need to create a seperate variable for it as its default is 10s  and I am not logging it as a warning

Reply:
The default may change (it's JVM specific), I think both should be logged in the same way

Reply:
ok, I will put warning for negative one too